pages/docs/community/styleguide/grammar.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Grammar
+description: This document provides grammar usage guidelines for clear and effective technical writing.
+---
+## Grammar
+
+This style guide defines standards for AsyncAPI documentation for grammar across topics such as abbreviations and acronyms, active voice, capitalization, spelling, and verb tense. If you are a part of the AsyncAPI Slack, use the #13_docs channel for any queries. Please make sure to go through the style guide before contributing to AsyncAPI documentation.
```

Comment:
```suggestion
This style guide defines standards for AsyncAPI documentation for grammar across topics such as abbreviations and acronyms, active voice, capitalization, spelling, and verb tense. 
```

I don't think we should mention the slack channel in this doc so I removed it. ✌🏽